src/Pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component, useEffect, useState } from 'react';
+import CardList from '../../components/CardList/CardList';
+import CreateCard from '../../components/CreateCard/CreateCard';
+import s from './MainPage.module.css';
+import { Link, NavLink, Outlet, redirect, useNavigate } from 'react-router-dom';
+import { CardProps } from '../../components/CardList/Card/Card';
+import { CardListDefault } from '../../components/CardList/TripsData';
+
+const MainPage = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [cardList, setCardList] = useState<CardProps[]>(CardListDefault);
+  const [windowUrl, setWindowUrl] = useState('./');
+  const navigate = useNavigate();
+  /* useEffect(() => {
+    navigate(windowUrl);
+  }, []);
+*/
+  const updateCardList = (card: CardProps) => {
+    console.log(card);
+    setCardList((prev) => [...prev, card]);
+  };
```

Comment:
we pass it to CreateCard component as a prop, and it's important to optimize component's performance using useCallback, 
const updateCardList =  useCallback((card: CardProps) => {
    console.log(card);
    setCardList((prev) => [...prev, card]);
  }, [])

Reply:
Understood, but I need check this theme later